docs/adrs/ADR-1.md

```diff
@@ -1,5 +1,4 @@
 # ADR - 1
-# THIS IS TEMPORARY -- WIll be making a valid ADR soon
 
 ## Title
```

Comment:
Maybe add a last-updated date here, so that people can know if things get very out of date. For example, if we start using a different Google Sheet, or if we switch to some UI -- we have to update those values in the ADRs. As the number of ADRs grow, I suspect this will get a bit hard.

Reply:
Makes sense! I was also apprehensive of adding the google sheet link as that might get outdated. I'll remove the link then.
If we add a UI, maybe that can come out as a separate small ADR